### PR TITLE
Allow country to be set using quickstart

### LINKED ISSUE #5796: COUNTRY not available via quick start
The flowdb synth data container allows one to specify a country to generate data for, but this isn't available when using the Quickstart script.

### PR DESCRIPTION
**Pull request by greenape**
Closes #5796 

### I have:

- [ ] Formatted any Python files with [black](https://github.com/ambv/black)
- [x] Brought the branch up to date with master
- [x] Added any relevant Github labels
- [ ] Added tests for any new additions
- [x] Added or updated any relevant documentation
- [ ] Added an Architectural Decision Record (ADR), if appropriate
- [ ] Added an [MPLv2 License Header](https://www.mozilla.org/en-US/MPL/headers/) if appropriate
- [x] Updated the [Changelog](https://github.com/Flowminder/FlowKit/blob/master/CHANGELOG.md) 

### Description

Allows setting the country used for synthetic data with quickstart script.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203803083784273

### CODE REVIEW

**Comment by jc-harrison** [comment]
For some reason when I tried this, it didn't start flowmachine. Probably a weird glitch on my end - I'll try again.

**Comment by flowstef** [comment]
Tried this but it still doesn't work for me. I can run the script but once it's up I still get `NPL` admin areas in the example notebooks.

**Comment by jc-harrison** [comment]
> Tried this but it still doesn't work for me. I can run the script but once it's up I still get `NPL` admin areas in the example notebooks.

@flowstef Note that to test this you'll need to export `GIT_REVISION` as the commit hash of the commit you want to test, otherwise it will use the compose files from master. Just pointing this out, because I forgot this step...

**Comment by flowstef** [comment]
I tried both with the GIT_REVISION var and running it just by checking out the correct branch and using the local `quick_start.sh` as opposed to the one from github.

**Comment by greenape** [comment]
Ah, my bad - the environment variables provided are being overridden by the downloaded `development_environment` ones.

**Review by jc-harrison — COMMENTED** [review]
Still failing to start flowmachine, but I can't see how that could be related to these changes. I'm getting errors when I run the script:
```
Starting containers (this may take a few minutes)
Pulling worked_examples          ... done
Pulling flowauth                 ... done
Pulling flowapi                  ... done
Pulling flowdb                   ... done
Pulling flowmachine_query_locker ... done
Pulling flowmachine              ... done
Creating network "flowkit_qs_default" with the default driver
Creating network "flowkit_qs_flowetl" with the default driver
Creating network "flowkit_qs_zero" with the default driver
Creating network "flowkit_qs_api" with the default driver
Creating network "flowkit_qs_db" with the default driver
Creating network "flowkit_qs_redis" with the default driver
Creating worked_examples ...
Creating flowdb          ...
Creating flowapi         ...
Creating flowauth        ...
Creating flowmachine_query_locker ...

ERROR: for flowmachine_query_locker  UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)

ERROR: for flowdb  UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)

ERROR: for flowauth  UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)

ERROR: for flowapi  UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)

ERROR: for worked_examples  UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)

ERROR: for flowmachine_query_locker  UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)

ERROR: for flowdb  UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)

ERROR: for flowauth  UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)

ERROR: for flowapi  UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)

ERROR: for worked_examples  UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)
ERROR: An HTTP request took too long to complete. Retry with --verbose to obtain debug information.
If you encounter this issue regularly because of slow network conditions, consider setting COMPOSE_HTTP_TIMEOUT to a higher value (current value: 60).
```
After waiting a while everything starts up OK despite the error messages, except that there's no sign of flowmachine.

But flowdb does have data for the specified country. I don't think the disaster region pcod is working, though - with `COUNTRY=GBR` and `DISASTER_REGION_PCOD=GBR.1.36_1`, looking at modal locations in the 2 weeks starting 2016-02-11 (the default disaster start date), there are still a normal number of residents in Greater London.
Screenshot 2023-01-26 at 12 00 31

**Comment by jc-harrison** [comment]
Ah, I just re-read the docs change and realised I should be using `EXAMPLE_COUNTRY` and `EXAMPLE_DISASTER_REGION_PCOD`. `COUNTRY` appeared to work as expected though, although `DISASTER_REGION_PCOD` didn't.

File: quick_start.sh [hunk 1]
```diff
@@ -129,6 +129,8 @@ then
     $DOCKER_COMPOSE down -v
 else
     source /dev/stdin <<< "$(curl -s https://raw.githubusercontent.com/Flowminder/FlowKit/${GIT_REVISION}/development_environment)"
+    export COUNTRY=${EXAMPLE_COUNTRY:-$COUNTRY}
+    export DISASTER_REGION_PCODE=${EXAMPLE_DISASTER_REGION_PCODE:-$DISASTER_REGION_PCODE}
```
Comment (jc-harrison) [inline quick_start.sh:133]:
```suggestion
    export DISASTER_REGION_PCODE=${EXAMPLE_DISASTER_REGION_PCOD:-$DISASTER_REGION_PCOD}
```

**Pull request merged**